ui/controls.js

```diff
@@ -18,14 +18,14 @@ goog.require('shaka.ui.AdPosition');
 goog.require('shaka.ui.BigPlayButton');
 goog.require('shaka.ui.Locales');
 goog.require('shaka.ui.Localization');
-goog.require('shaka.ui.SeekBar');
 goog.require('shaka.ui.Utils');
 goog.require('shaka.util.Dom');
 goog.require('shaka.util.EventManager');
 goog.require('shaka.util.FakeEvent');
 goog.require('shaka.util.FakeEventTarget');
 goog.require('shaka.util.IDestroyable');
 goog.require('shaka.util.Timer');
+goog.require('shaka.ui.SeekBarFactory');
```

Comment:
When you rename this to `SeekBar.Factory`, you should put back `goog.require('shaka.ui.SeekBar')` and put it back in alphabetical order.

Reply:
Done